verify_installation.py

```
@@ -33,22 +38,26 @@ def check_sleap_installation():
-        
+
+        print(
+            f"[OK] sleap-roots-training imported successfully (version: {sleap_roots_training.__version__})"
+        )
+
         # Check all modules
-        modules = ['config', 'train', 'evaluate', 'models', 'datasets']
+        modules = ["config", "train", "evaluate", "models", "datasets"]
         for module in modules:
             try:
                 mod = getattr(sleap_roots_training, module)
```

Ruff (0.12.2)

Local variable `mod` is assigned to but never used

Remove assignment to unused variable `mod`

(F841)